mkosi

```diff
@@ -3498,6 +3511,44 @@ def run_qemu(args):
 
     os.execvp(cmdline[0], cmdline)
 
+def expand_paths(input_paths):
+    if not input_paths:
+        return []
+
+    environ = os.environ.copy()
+    # Add a fake SUDO_HOME variable to allow non-root users specify
+    # paths in their home when using mkosi via sudo.
+    sudo_user = os.getenv("SUDO_USER")
+    if sudo_user:
+        environ["SUDO_HOME"] = os.path.expanduser("~{}".format(sudo_user))
+
+    # Don't use os.path.expandvars because it treats unset variables
+    # as empty. Here entries that refer to unset variables are ignored.
+    vars_pattern = re.compile(r'\$(\w+|\{[^}]*\})', re.ASCII)
```

Comment:
urks... regexes... 

Why not just use python template strings? i.e. this stuff: https://www.python.org/dev/peps/pep-0292/

Reply:
No reason except I forgot about those, just changed to use it.